examples/acados_python/zoRO_example/diff_drive/main.py

```diff
@@ -26,24 +26,25 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.;
 
-import os
 
-import numpy as np
-import matplotlib.pyplot as plt
+# TODO:
+# - beta only diagonal
```

Comment:
Has 'beta only diagonal' been implemented? Line 470-478 in zoro_template.in.c

Reply:
Yes!